src/components/Tables/Table.tsx

```diff
@@ -206,7 +189,7 @@ const Table: React.FC<Props> = ({
         return {
             name: headerName,
             selector: (row) => row.data[headerKey] ?? "",
-            sortable: sortable,
+            sortable: sortable?.some((sortKey) => sortKey === headerKey),
```

Comment:
Do we need a description of what this is in a comment or no? Tbf it's easy to understand but did need a google from me

Reply:
I reckon it's probably fine